src/utils/mkarray.h

```diff
@@ -44,6 +49,26 @@ inline mxArray* make_mx_array(const std::vector<std::string> &v) {
 	return data;
 }
 
+template<>
+inline mxArray* make_mx_array(const std::vector<nix::Value> &v) {
+	if (v.empty()) {
+		return nullptr;
+	}
+
+	nix::DataType ntype = v[0].type();
+	mxArray *data;
+	if (ntype == nix::DataType::Nothing) {
+		data = nullptr; //fixme: something else?
+	} else {
+		data = mxCreateCellMatrix(1, v.size());
+		for (size_t i = 0; i < v.size(); i++) {
+			mxSetCell(data, i, make_mx_array(v[i]));
+		}
+	}
+
+	return data;
+}
+
```

Comment:
Well, I just thought we keep in "utils" only things that we need in several places. We probably need to convert vector<Value> -> mxArray only for nixsection, and only once. Same for Value -> mxArray actually. So IMHO would be nicer o keep them in the nixsection, as they are just locally used, what do you think?

And if we keep vector <nix::Value> in mkarray then logically we should move vector<Tag>, ... etc also there, which would be a mess, right?)

But it could also stay like this. If you say you'd prefer it better, I just merge it - really no problem.